### PR TITLE
add: rules page (#3)

### LINKED ISSUE #3: Add Rules Page.
The current link to [the rules page](https://talks.ourtech.community/rules) on the Talks web site home page results in a 404.

We should add the rules mentioned in [the CFP](https://talks.ourtech.community/cfp) there.

### PR DESCRIPTION
**Pull request by tusharnankani**
Closes #3.

### CODE REVIEW

File: rules.html [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+					</li>
+				</ul>
+			</nav>
+		</header>
+
+		<h1>OTC Talks Rules</h1>
+
+		<main id="main-content">
+			<ul>
+				<li>The mode of OTC Talks is Online.</li>
```
Comment (HarshKapadia2) [inline rules.html]: A better way to frame it in my opinion: 'OTC Talks will be conducted online.'

File: rules.html [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+				<li>
+					Duration of the lightning talk: 15 - 25 minutes from 10:45
+					PM IST on a Saturday, followed by the CatchUp on the
```
Comment (HarshKapadia2) [inline rules.html]: `CatchUp` should be `OTC CatchUp` and should be hyperlinked to the OTC CatchUp home page. (I'm not sure whether it should be opened in a new page or on the same page itself.)

File: rules.html [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+					>
+					while attending our talks. Any violations can be reported to
+					<a
+						href="https://ourtech.community/team"
+						target="_blank"
+						rel="noreferrer"
+						>the organisers</a
+					>
+					or on
+					<a href="mailto:contact@ourtech.community"
```
Comment (HarshKapadia2) [inline rules.html]: This `a` tag should have a `rel` attribute with the value `noreferrer`.

**Review by HarshKapadia2 — CHANGES_REQUESTED** [review]
Other than a few small changes, this seems good to me. Thank you so much, @tusharnankani!

**Comment by KartikSoneji** [comment]
@tusharnankani can you please also change
```html
<a
    href="..."
>Abcd</a>
```

To
```html
<a href="...">
    Abcd
</a>
```

Prettier by default cares about whitespace sensitivity for XML, but this should be fine in HTML.

**Comment by tusharnankani** [comment]
@KartikSoneji Wait what? We do not want the changes prettier is making? Then why are we using it 😭

FYI: It is that way in EVERY HTML file of EVERY repository of OTC.

File: rules.html (hunk 1 above)
Comment (tusharnankani) [reply rules.html]: Fixing it.

File: rules.html (hunk 2 above)
Comment (tusharnankani) [reply rules.html]: Yep.

File: rules.html (hunk 3 above)
Comment (tusharnankani) [reply rules.html]: https://github.com/OurTechCommunity/meetup/blob/a8d7ea90c0ac5bc455d9576e27590d18857e72e7/rules.html#L144-L146

*coughs*

Comment (tusharnankani) [reply rules.html]: Fixed this though.

**Comment by KartikSoneji** [comment]
> @KartikSoneji Wait what? We do not want the changes prettier is making? Then why are we using it 😭

We can change this with `html-whitespace-sensitivity` https://prettier.io/docs/en/options.html#html-whitespace-sensitivity

File: rules.html (hunk 3 above)
Comment (HarshKapadia2) [reply rules.html]: Yeah the e-mail in the OTC MeetUp rules should be fixed. Thank you for fixing this though.

**Comment by HarshKapadia2** [comment]
> We can change this with `html-whitespace-sensitivity`

Let us do that later. Also, the Prettier config file is the same in all repositories across the organisation, so let us not change it for now. If we want to make changes, we should implement some global store for these things.

The more we build, the more I am noticing the requirement for having global standards and just using those. What do we do @KartikSoneji. We should have a discussion on this. (Can you please raise an issue on the OTC CatchUp repo regarding this? Should we use the Discussions tab for this rather than an issue?)

For now though, this PR looks good to me. I'll approve and merge it.

**Review by HarshKapadia2 — APPROVED** [review]
Looks good to me! Thank you for all the changes, @tusharnankani!

**Pull request merged**